demosplan/DemosPlanCoreBundle/Entity/Permission/AccessControlPermission.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Entity\Permission;
+
+use DateTime;
+use DemosEurope\DemosplanAddon\Contracts\Entities\AccessControlPermissionInterface;
+use DemosEurope\DemosplanAddon\Contracts\Entities\CustomerInterface;
+use DemosEurope\DemosplanAddon\Contracts\Entities\OrgaInterface;
+use DemosEurope\DemosplanAddon\Contracts\Entities\RoleInterface;
+use DemosEurope\DemosplanAddon\Contracts\Entities\UuidEntityInterface;
+use demosplan\DemosPlanCoreBundle\Entity\CoreEntity;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * Stores the information about the current phase of a procedure.
```

Comment:
Naming:
In the context of a procedure, a phase is one of the listed in procedure_phases.yml.
So this naming is -at least to me- confusing. 
What is meant with "phases" in this context?
But Im not fully understand, what this Entity

Reply:
Updated in new version